ivy/functional/ivy/elementwise.py

```diff
@@ -3796,7 +3796,7 @@ def not_equal(
     Functional Examples
     ------------------
 
-    With :code:`ivy.Array` inputs:
+    With: code:`ivy.Array` inputs:
```

Comment:
The colons should surround 'code' as before, like `With :code:`ivy.Array` inputs:`

Reply:
hi could you pleas assist me why then docstring test is failing